pkgs/by-name/ri/rime-ice/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "rime-ice";
+  version = "2024.05.21";
+
+  src = fetchFromGitHub {
+    owner = "iDvel";
+    repo = "rime-ice";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-lpPwGzuAAAzJBgcVW0o5z4lzAzKUOuo7Ae6ySltzhVE=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/rime-data
+    cp -r ./* $out/share/rime-data
```

Comment:
Presumably go source files should not be installed.  There probably are other files that should not be installed.

Reply:
I am not sure which parts are necessary and which are unnecessary; I need to ask upstream first to accurately distinguish between them

Reply:
I rm `/others` now.